arches/install/arches-templates/project_name/settings.py-tpl

```diff
@@ -139,7 +139,7 @@ INSTALLED_APPS = (
     "arches.app.models",
     "arches.management",
     "guardian",
-    "captcha",
+    "django_recaptcha",
```

Comment:
see https://github.com/django-recaptcha/django-recaptcha/blob/main/CHANGELOG.md#upgrade-considerations-from-v3-to-v4